bin/picom-trans

```diff
@@ -280,8 +278,10 @@ set_opacity()
   # strip trailing '%' sign, if any
   target_opacity="${target_opacity%%%}"
 
-  current_opacity=$(get_current_opacity)
-  echo "$target_opacity" | grep -qE '^[+-]' \
+  current_opacity="$(get_current_opacity)"
+  test -z "$current_opacity" && current_opacity=100
```

Comment:
`get_current_opacity` has changed and now returns the opacity as a 32bit integer, not in range 0 - 100. Either convert it first (both in range 0 - 100) or convert the target opacity here and not later (both as 32bit integer).
This breaks incrementing/decrementing the opacity with `±5`.
Furthermore, there is no need to get the current opacity if we are going to override it anyway (i.e. no inc/dec).

Reply:
d3b29fc